node/src/components/deploy_buffer.rs

```diff
@@ -85,7 +84,7 @@ impl Display for Event {
 }
 
 type DeployCollection = HashMap<DeployHash, DeployHeader>;
-type ProtoBlockCollection = HashMap<ProtoBlockHash, DeployCollection>;
+pub type ProtoBlockCollection = HashMap<ProtoBlockHash, DeployCollection>;
```

Comment:
I don't like making it pub type but I understand the alternative will be a mouthful.